typescript/web/src/components/export-button/export-modal/index.tsx

```diff
@@ -104,8 +104,10 @@ const exportDataset = async ({
   const blobDataset = await (await fetch(exportDatasetUrl)).blob();
   const url = window.URL.createObjectURL(blobDataset);
   const element = document.createElement("a");
+  const extension =
+    format === ExportFormat.Yolo || options.coco?.exportImages ? "zip" : "json";
```

Comment:
To be more future-proof, I'd have factorized things a little bit. But I doubt it's really useful since me might remove this part once we drop the SW.
Did you have time to understand why it downloaded the file in a blob or asked old devs?

Reply:
Yes I was about to refactor all that, but thought that it would be a waste of time since it should probably get re-refactored again soon.

No I didn't ask old devs, but I understand the reason: it's to obfuscate the URL to the download API that creates the signed URL: it would allow getting new signed URLs anytime, since the file is never deleted from our bucket.
See that comment: https://github.com/labelflow/labelflow/issues/751#issuecomment-1012074286